Makefile

```diff
@@ -12,9 +12,14 @@ PROTOC=$(DOCKER) run --rm -v $(shell pwd):/mnt $(PROTOC_IMAGE)
 CLIENT_JARS_BUCKET="s3://treeverse-clients-us-east/"
 
 # https://openapi-generator.tech
-OPENAPI_GENERATOR_IMAGE=openapitools/openapi-generator-cli:v5.3.0
+OPENAPI_GENERATOR_VERSION=v5.3.0
+OPENAPI_GENERATOR_IMAGE=openapitools/openapi-generator-cli:$(OPENAPI_GENERATOR_VERSION)
 OPENAPI_GENERATOR=$(DOCKER) run --user $(UID_GID) --rm -v $(shell pwd):/mnt $(OPENAPI_GENERATOR_IMAGE)
 
+OPENAPI_GENERATOR_VERSION_ADVANCED=v7.0.0
+OPENAPI_GENERATOR_IMAGE_ADVANCED=openapitools/openapi-generator-cli:$(OPENAPI_GENERATOR_VERSION_ADVANCED)
+OPENAPI_GENERATOR_ADVANCED=$(DOCKER) run --user $(UID_GID) --rm -v $(shell pwd):/mnt $(OPENAPI_GENERATOR_IMAGE_ADVANCED)
```

Comment:
The more I think about these names the less I am a fan of them.  Perhaps call them "OPENAPI_GENERATOR_SERVER" and "OPENAPI_GENERATOR_CLIENT_SDK"?  (Extremely unimportant, any which way you decide)

Reply:
the only difference between them is the openAPI version they use. This is the reason that I've added the `ADVANCED` part to the newer version. 
They both generate the client's SDK